packages/runtime/src/runtime.ts

```diff
@@ -433,6 +433,17 @@ export class Runtime {
     this.ctx.state.accounts.set(senderAcc.address, senderAcc);
     this.ctx.state.globalApps.set(app.id, senderAcc.address);
 
+    for (const [address, account] of this.store.accounts) {
```

Comment:
why do we need it here ?

Reply:
we already have this in initializeAccounts or is there any other reason ?

Reply:
> why do we need it here ?

Because earlier we were only preparing `ctx` in senderAcc. But we should encapsulate all created assets/apps before run. Because if an opcode during deploy requires information from external account, then that account should be present in `ctx` as well. Please let me know if that makes sense.

> we already have this in initializeAccounts or is there any other reason ?

In `initializeAccounts` we are preparing store, here we are preparing `ctx`(before run).

Reply:
how about removing lines 436 to 446.
and add `this.ctx.state = cloneDeep(this.store);` at 430.
This does the same thing and is easily understandable.

Checked on my system test run properly.

Reply:
yes, good idea